src/main/java/httpserver/http/response/ResponseBuilder.java

```diff
@@ -4,36 +4,19 @@
 import httpserver.http.StatusCode;
 
 public class ResponseBuilder {
```

Comment:
I think this type of class would normally be consdiered a `Factory` rather than a `Builder`.